charts/mercure/templates/deployment.yaml

```diff
@@ -47,6 +47,11 @@ spec:
                 configMapKeyRef:
                   name: {{ include "mercure.fullname" . }}
                   key: global-options
+            - name: EXTRA_DIRECTIVES
+              valueFrom:
+                configMapKeyRef:
+                  name: {{ include "mercure.fullname" . }}
+                  key: extra-directives
```

Comment:
The config template must be updated as well isn't it? We also need a new value.

Also, I would use a secret as this env var may contains secret keys and things like this, depending on the modules you use.

WDYT?

Reply:
I did it, but it may be confusing with `extra-directives`/`extraDirectives`, should I change it too?